docs/lib/ssr/fragments/js/getting-started.md

```diff
@@ -0,0 +1,106 @@
+To work well with server-rendered pages, Amplify JS requires slight modifications from how you would use it in a client-only environment.
+
+## Amplify
+
+### Enabling SSR 
+
+When using the Amplify CLI, the __aws-exports.js__ file gets created and updated automatically for you based upon the resources you have added and configured.
+
+For client-only apps, `Amplify.configure(awsExports)` is all you need.
+
+To enable SSR support, also provide `ssr: true`:
+
+```js
+import { Amplify } from "aws-amplify"
+import awsExports from "../src/aws-exports"
+
+Amplify.configure({ ...awsExports, ssr: true })
+```
+
+> **Note**: Once [vercel/next.js#16977](https://github.com/vercel/next.js/issues/16977) is resolved, you can hoist `Amplify.configure` into **pages/_app.js**.  Until then, be sure that all **pages/*** run `Amplify.configure({ ...awsExports, ssr: true })`.
+
+
+### withSSRContext
+
+Your page components should continue using top-level category imports for client-side code:
+
+```js
+import { Amplify, API } from "aws-amplify"
+
+export default function HomePage({ posts = [] }) {
+  const [posts, setPosts] = useState(posts)
+
+  useEffect(() => {
+    // 👇 Notice how the client correctly uses the top-level `API`
+    API.graphql({ query: listPosts }).then(({ data }) => setPosts(data.listPosts.items))
+  }, [])
+
+  return ( ... )
+}
+```
+
+However, when in server-only functions (e.g. Next.js' `getServerSideProps`, `getStaticProps`, & `getStaticPaths`), use `withSSRContext({ req?: ServerRequest })`:
+
+```js
+import { Amplify, API, withSSRContext } from "aws-amplify"
```

Comment:
Should API and Amplify still be imported here?

Reply:
This is a far callout since we are pulling `API` from `SSR` now. I do believe the `Amplify` is there @dabit3 for the `...` of the `aws-exports`

Reply:
Resolved via our call. Top-level imports like `API` should still be used in client-side code.